examples/complete-http/main.tf

```diff
@@ -304,7 +304,6 @@ resource "tls_private_key" "private_key" {
 }
 
 resource "tls_self_signed_cert" "example" {
-  key_algorithm     = tls_private_key.private_key.algorithm
```

Comment:
This is unrelated to the changes I have made, but needed to be removed to allow the pre-commit hooks to be completed successfully.  I am assuming that the attribute was changed to read only in a version of the provider released after the example was written!

https://registry.terraform.io/providers/hashicorp/tls/latest/docs/resources/self_signed_cert#key_algorithm

Reply:
You are right, thank you! Related issue in the TLS provider - https://github.com/hashicorp/terraform-provider-tls/issues/174